KinMigrationModule/KinMigrationModule/KinMigrationManager.swift

```diff
@@ -170,10 +182,18 @@ extension KinMigrationManager {
             return
         }
 
-        biDelegate?.kinMigrationCallbackStart()
-        delegate?.kinMigrationManagerDidStart(self)
-
-        startBurningAccount(account)
+        shouldMigrate()
```

Comment:
@alongenosar you need to cleanup the migration if an error is thrown.

Reply:
done